src/azure/cli/tests/test_generic_update.py

```diff
@@ -14,6 +14,7 @@
 
 #pylint:disable=invalid-sequence-index
 #pylint:disable=unsubscriptable-object
+#pylint:disable=line-too-long
```

Comment:
there are only 2 lines over limit, we might as well break them and avoid this disable